.gitIgnore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Your `.gitignore` file is not working and the `test.md` file was not ignored and committed to this repository. Please, double-check for typos in the name of your files and make sure the `test.md` file is not committed to this repository.

Reply:
ok but do i close the pull request in order to make the changes